src/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ add_subdirectory(data)
 add_subdirectory(opengl)
 add_subdirectory(util)
 add_subdirectory(xdl)
+add_subdirectory(CAD)
```

Comment:
The CAD directory was never properly imported in this branch, so trying to include `grail-CAD` in `grail-path` just added an empty string.